### ISSUE TITLE
Promote distribution alternatives for ease of access & personal development styles

### ISSUE THREAD

**Issue by Martin-Pitt** [Popmotion/popmotion#212]
As someone who doesn't use build tools, it was frustrating when I initially came to popmotion, as I struggled to understand how to start.

I jumped from the site to the github repo and thought
> Ugh, do I really have to setup a build? I know webpack but that's a lot of effort for a library that might not even fit my usecase. I hope there is a classic `dist` folder and I can figure out what file(s) I can get away with.

Soo… ever since I've been manually downloading `dist/popmotion.global.min.js` from the master branch here on github, which I _think_ is the right one for most purposes? (Let alone the confusion of how stylefire comes in, which appears as a separate library from the website docs)

----

I feel that not having a non-commandline/non-build option adds a ton of friction to new users.

Doesn't feel like there is a way to just _get up and go_.

On top of that, everyone has their own style of doing web development. Sometimes even internal guidelines that they'd prefer to stick with consistently to make it easy on their colleagues.

So what I'd like to see is a README.md & updated site:

- Instructions with a script tag for a local file / complemented by a direct download
- Instructions with using the new EcmaScript Module syntax (I think it was `<script type="module" …` and then a snippet for the import syntax?)
- Instructions with a script tag to a CDN version
  - https://cdnjs.com/libraries/popmotion (Huh, is this outdated by a major version?)
  - https://www.jsdelivr.com/
  - …?
- Instructions that lead you directly to a online / collaborative editor example you can fork / play with
  - There's a lot of options here, but I think best stick with CodePen & JSBin as they contrast eachother nicely
  - An additional option for the build tool side of things, might be WebPackBin – it's really great, as you can get a webpack project going fast and configured easily using a GUI setup, when you are ready to move onto local dev, you can download the entire project

Basically I should be able to view all of my options at a glance when wanting to try popmotion or download the latest.

**Comment by InventingWithMonster** [comment]
Hey man! Excellent point.

I'm overhauling the Popmotion website as we speak. I've just added a new page to the Learn section, which will be linked to from README.md and the "Get started" page:

```
---
title: Install Popmotion
description: Overview of Popmotion's different installation options.
category: basics
---

# Install Popmotion

## Package managers

### npm

```bash
npm install popmotion --save
```

### yarn

```bash
yarn add popmotion
```

## File include

**Note:** The Popmotion docs use the `import` syntax for importing individual modules. If you use one of the following installation methods, Popmotion will be available on the global `popmotion` variable.

So when you see in the docs `import { tween } from 'popmotion'`, you can use `popmotion.tween` instead.

### Download

You can download the latest version of Popmotion at https://unpkg.com/popmotion/dist/popmotion.global.min.js

### `script` include::

Or include it directly in your HTML with this `script` tag:

```html
<script src="https://unpkg.com/popmotion/dist/popmotion.global.min.js"></script>
```

### CodePen

[This CodePen](https://codepen.io/popmotion/pen/zPjXWa?editors=0010) links to a Popmotion playground, which is set up with the latest version of Popmotion.

You an also add Popmotion to any CodePen project by clicking on Settings > JavaScript and then pasting `https://unpkg.com/popmotion/dist/popmotion.global.min.js` into the "Add External JavaScript" field.
```

**Comment by jimaek** [comment]
With jsDelivr you will also be able to get download stats for your project, and it also pulls from npm https://www.jsdelivr.com/package/npm/popmotion
http://cdn.jsdelivr.net/npm/popmotion@7.8.2/dist/popmotion.global.min.js

**Comment by InventingWithMonster** [comment]
@jimaek I'll take a look at that, thanks for the heads up!

**Comment by Martin-Pitt** [comment]
Just thinking around; Have been trying to sketch a cut down version.
I think we can simplify it down to their installation & usage across 3 common usecases of modules vs global.


As a Module via Bundler or Node.js
=================

Install Popmotion
-----------------
```bash
npm install popmotion --save
```
```bash
yarn add popmotion
```

Import Module via Bundler
--------------------------
```js
import { tween } from 'popmotion'
```

Import Module via Node.js
--------------------------
```js
const { tween } = require('popmotion');
```

As a Global in the Browser
===================

Download the latest at: https://unpkg.com/popmotion/dist/popmotion.global.min.js

Or alternatively for non-production:
```html
<script src="https://unpkg.com/popmotion/dist/popmotion.global.min.js"></script>
<script>
   popmotion.tween({ onUpdate: (x) => console.log(x) }).start();
</script>
```

As a Module in the Browser
====================
via [ES6 modules](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import):
```html
<script type="module">
   import { tween } from 'https://unpkg.com/popmotion/dist/popmotion.global.min.js';
</script>
```
